JobConfig/digitize/prolog.fcl

```diff
@@ -159,6 +159,7 @@ Digitize: {
     "keep mu2e::CaloClusters_*_*_*",
     "keep mu2e::TriggerInfo_*_*_*",
     "keep mu2e::KalSeeds_Trig*_*_*",
+    "keep mu2e::KalSeeds_TT*_*_*",
     "keep mu2e::KalSeedMCs_TT*KSFMC_*_*",
```

Comment:
Should the KalSeedMCs search string match the KalSeed?

Reply:
It probably should, I just didn't update its naming style since it only exists in Production. I can put this renaming in this PR as well if you would like